lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -70,6 +71,10 @@ def location
         def message(_node = nil)
           format(MSG, preferred: preferred_name, bad: variable_name)
         end
+
+        def splat_exceptions?(location)
+          location.source.to_s.start_with?('*')
```

Comment:
It's best to check the type of the node instead of the source code, and filter out stuff based on it. Other things that should be filtered out are constants and method calls with params, that are producing some list of exceptions.